Sources/AppcuesKit/Presentation/UI/Components/MultilineTextView.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  MultilineTextView.swift
+//  AppcuesKit
+//
+//  Created by Matt on 2022-09-12.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+internal struct MultilineTextView: UIViewRepresentable {
+    @Binding var text: String
+    let model: ExperienceComponent.TextInputModel
+
+    func makeCoordinator() -> Coordinator {
+        Coordinator(self)
+    }
+
+    func makeUIView(context: Context) -> UIView {
+        if (model.numberOfLines ?? 1) > 1 {
+            return makeTextView(context: context)
+        } else {
+            return makeTextField(context: context)
+        }
+    }
```

Comment:
All I can say is wow! impressive wrapper here with conditional switching between UITextView and UITextField.  Unfortunate all the work is required, but this seems great how you pulled it off.